versioned_docs/version-2.x/magento1/installation.mdx

```diff
@@ -217,9 +219,30 @@ For more security add a random key on `frontcommerce_secret_key` in your
     // ...
 ```
 
+### Configure facet order
+
+<SinceVersion tag="2.19" />
```

Comment:
2.19 ? is this planned to be backported ? Also, [according to this page](https://www.notion.so/frontcommerce/Releases-a345faecf6624ce6bd91623f8580b16e?pvs=4#d9420f41fe5749d48b652318cdd257e0), 2.19 support ended with 3.0.0 if so, this should not be a `feat` commit and this MR should not target `next` but `main`

Reply:
Nope, it was a copy-paste fail...

Reply:
Fixed